### DISCUSSION TITLE
Attributes with character ':'

### DISCUSSION THREAD

**Discussion by CleitonDeLima** [django-components/django-components#556] [Q&A] [answered]
Let's say I have an input component, how could I enter the `:placeholder="foo"` or `x-bind:placeholder="foo"` attribute in the component?

If I inform `:` indicates that I am informing a dict to the component, what would be an alternative?

**Comment by JuroOravec** [comment, marked as answer]
Hey @CleitonDeLima, the first format `:placeholder="foo"` should work. The input is converted to a dict only if there is a prefix before the `:`. But if the key starts with `:`, so `:placeholder="foo"`, this would remain unchanged. On the other hand, `x-bind:placeholder="foo"` would be converted to `x-bind={"placeholder": "foo"}`.

**Reply by JuroOravec** [reply]
Ah, I just realized I wrote the logic for permitting attributes that start with colon (like `:placeholder="foo"`), but didn't release it yet. It should be ready in about an hour.

**Reply by JuroOravec** [reply]
Alright, the fix was released as part of [v0.85](https://github.com/EmilStenstrom/django-components/releases/tag/0.85)
